### PR TITLE
Test filter task

### PR DESCRIPTION
**Pull request by esmaoui**

### CODE REVIEW

File: gatein/src/test/java/org/exoplatform/platform/qa/ui/gatein/GateinUserAndGroupManagementTestIT.java [hunk 1]
```diff
@@ -347,6 +348,7 @@ public void test11_SearchUserFromHomeSearchBar() {
 
   @Test
   @Tag("search")
+  @Tag("aymen")
```
Comment (ilyeskallala) [inline gatein/src/test/java/org/exoplatform/platform/qa/ui/gatein/GateinUserAndGroupManagementTestIT.java]: useless Tag

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/FilterTaskTestIT.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        info("Start setUpBeforeMethod");
+        homePagePlatform = new HomePagePlatform(this);
+        tasksManagement = new TasksManagement(this);
+        projectsManagement = new ProjectsManagement(this);
+        manageLogInOut = new ManageLogInOut(this);
+        manageLogInOut.signInCas(PLFData.USER_ROOT,PLFData.password);
+
+    }
+   /*
+    bug TA-571
```
Comment (ilyeskallala) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/FilterTaskTestIT.java]: use @tag("") to define the issue number

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/wiki/WikiLocators.java [hunk 3]
```diff
@@ -500,10 +500,10 @@
                                                                                 By.xpath(".//*[@id='UIWikiPageEditForm']//*[contains(text(),'Draft saved')]");
 
   public static final SelenideElement ELEMENT_SAVE_BUTTON_ADD_PAGE                               =
-                                                                   $(byId("UISubmitToolBarUpper_SavePage_"));
+                                                                   $(byId("UISubmitToolBarBottom_SavePage_"));
 
   public static final By              ELEMENT_CANCEL_BUTTON_ADD_PAGE                             =
-                                                                     By.id("UISubmitToolBarBottom_Cancel_");
+                                                                     By.id("UISubmitToolBarUpper_Cancel_");
```
Comment (ilyeskallala) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/wiki/WikiLocators.java]: 1.why did you change this locator ? it may mess other tests

File: gatein/src/test/java/org/exoplatform/platform/qa/ui/gatein/GateinUserAndGroupManagementTestIT.java [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 import static com.codeborne.selenide.Selectors.*;
 import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.switchTo;
```
Comment (thomasdelhomenie) [inline gatein/src/test/java/org/exoplatform/platform/qa/ui/gatein/GateinUserAndGroupManagementTestIT.java]: Useless import ?

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/FilterTaskTestIT.java [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+        info("Start setUpBeforeMethod");
+        homePagePlatform = new HomePagePlatform(this);
+        tasksManagement = new TasksManagement(this);
+        projectsManagement = new ProjectsManagement(this);
+        manageLogInOut = new ManageLogInOut(this);
+        manageLogInOut.signInCas(PLFData.USER_ROOT,PLFData.password);
+
+    }
+
+    @Tag("TA-571")
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/FilterTaskTestIT.java]: Do we add a tag with the issue number now for each issue ?

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/taskmanagement/TaskManagementLocator.java [hunk 6]
```diff
@@ -69,6 +69,7 @@
   public static final SelenideElement ELEMENT_INPUT_LABEL                           = $(byClassName("addLabelInput"));
 
   public static final SelenideElement ELEMENT_LABEL_BUTTON_CONFIRM_DELETE           = $(byClassName("confirmDelete"));
+  public static final By ELEMENT_CLICK  = By.id("taskManagement");
```
Comment (thomasdelhomenie) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/taskmanagement/TaskManagementLocator.java]: The name of the variable is too vague IMO.

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/NavigationToolbar.java [hunk 7]
```diff
@@ -23,6 +23,7 @@
 import static com.codeborne.selenide.Selectors.byText;
 import static com.codeborne.selenide.Selectors.byXpath;
 import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.switchTo;
```
Comment (thomasdelhomenie) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/NavigationToolbar.java]: Useless import ?

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/FilterTaskTestIT.java (hunk 5 above)
Comment (esmaoui) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/FilterTaskTestIT.java]: Yes to know which issue reported has been automated.

File: gatein/src/test/java/org/exoplatform/platform/qa/ui/gatein/GateinUserAndGroupManagementTestIT.java (hunk 4 above)
Comment (esmaoui) [reply gatein/src/test/java/org/exoplatform/platform/qa/ui/gatein/GateinUserAndGroupManagementTestIT.java]: Yes

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/FilterTaskTestIT.java (hunk 5 above)
Comment (thomasdelhomenie) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/FilterTaskTestIT.java]: Can you show me how you use this tag ? Is there a report somewhere ?

Comment (aboughdiri) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/FilterTaskTestIT.java]: Yes we just mention the issue number to the test case, so we have a direct link from the code to the origin of the automated case so we can have a reference as specification for later re-use of question about from where we specified the expected behavior.
In addition of setting the PR on the issue to have a link from both sides.
